gnovm/tests/files/recover14.gno

```diff
@@ -0,0 +1,15 @@
+package main
```

Comment:
I think we should consider this case too.

```
package main


func main() {
     defer func() {
    		r := recover()
    		println("recover:", r)
    }()
	println(1/0)
}

```

We cannot recover it but maybe this should be failed on the preprocess

Reply:
this is not a runtime panic and type check already handle this.

Reply:
My bad; we can close it since it's already handled.